RockLib.Logging/CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.1.1 - 2024-07-16
+
+#### Changed
+RockLib.Configuration.4.0.0 -> RockLib.Configuration.4.0.1 to fix vulnerability
```

Comment:
Should be RockLib.Diagnostics 3.0.1 right?

Reply:
ah youre right